modules/features/falcon_feature_webform/config/install/field.field.paragraph.webform.field_webform.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_webform
+    - paragraphs.paragraphs_type.webform
+  module:
+    - webform
+id: paragraph.webform.field_webform
+field_name: field_webform
+entity_type: paragraph
+bundle: webform
+label: Webform
+description: 'Create new webform or edit existing webform you can on <a target="_blank" href="/admin/structure/webform">Webforms</a> page.'
```

Comment:
To create a new webform, go to <a target="_blank" href="/admin/structure/webform">Webforms</a> page.

Reply:
Changed.